src/components/CityCoinTxList.js

```diff
@@ -164,11 +196,24 @@ function uintJsonToRewardCycle(value) {
   );
 }
 
-function listCvToMiningAmounts(value) {
-  const amounts = value.repr;
-  // const amountsFiltered = amounts.split(' ');
-
-  return <>Amounts: {value.repr}</>;
+function listCvToMiningAmounts(value, blockHeight) {
+  const amountsJSON = cvToJSON(hexToCV(value.hex));
```

Comment:
CvToJson might fail if the amount is too high.

Reply:
Do you want to show the total?

Reply:
So far I haven't seen any issues with it failing and txs up to the full 200 supported, but I agree showing the total here is a good idea. Fixed in 0f1094b.